var/spack/repos/builtin/packages/libgcrypt/package.py

```diff
@@ -38,6 +38,10 @@ def flag_handler(self, name, flags):
         # flags, and the build system ensures that
         return (None, flags, None)
 
+    # 1.10.2 fails on macOS when trying to use the Linux getrandom() call
+    # https://dev.gnupg.org/T6442
+    patch("rndgetentropy_no_getrandom.patch", when="@=1.10.2 platform=darwin")
```

Comment:
Should we be using `@=` for all single-version when-statements now? Also, I'm curious if this patch could be applied to older versions or not.